features/filesystem/littlefs/littlefs/lfs_util.c

```diff
@@ -6,11 +6,19 @@
  */
 #include "lfs_util.h"
 
+
 // Only compile if user does not provide custom config
 #ifndef LFS_CONFIG
 
+#ifdef __MBED__
```

Comment:
nit: Could this be moved to the header file? And just make the crc function declared LittleFileSystem.cpp named "lfs_crc"?